numba/extending.py

```diff
@@ -165,18 +167,24 @@ def get(arr):
     from .typing.templates import make_overload_attribute_template
 
     def decorate(overload_func):
-        template = make_overload_attribute_template(typ, attr, overload_func)
+        template = make_overload_attribute_template(
+            typ, attr, overload_func,
+            inline=kwargs.get('inline', 'never'),
```

Comment:
This is just a minor nitpick, but maybe we could define a file somewhere `defaults.py` and put all our defaults in there. E.g. `DEFAULT_INLINE = 'never'`. This would a) add a central place for developers to lookup the defaults and b) make the code somewhat safer since a misspelt 'never' won't fail until runtime whereas a misspelt variable name like `DEFAULT_INLINE` can be caught by the syntax checker.

Reply:
I'd suggest doing this change in Q1 housekeeping.

Reply:
Also, That string will eventually be passed to `InlineOptions` which will do the check.  But, I agree that a central place for default values is a good idea.

Reply:
I think for internal use enums are good, but it's not necessarily something users would want so am inclined to keep a split?

Reply:
I don't think it's for user facing, but for internal code. Instead of spelling `"never"` on every API that have that as the default for inline, reference it as the variable. So it is easier to change if we ever need to.